spec/libsass/units/conversion/resolution/options.yml

```diff
@@ -0,0 +1,5 @@
+---
+:start_version: '3.4'
```

Comment:
Why are you adding these start versions? I don't think it makes sense to have start versions for the oldest supported language version...